pkg/decoder/tagxl/v1/port152.go

```diff
@@ -6,17 +6,65 @@ import (
 	"github.com/truvami/decoder/pkg/decoder"
 )
 
+// Version: 1
+// +------+------+-----------------------------------------------+------------+
+// | Byte | Size | Description                                   | Format     |
+// +------+------+-----------------------------------------------+------------+
+// | 0    | 1    | version                                       | uint8      |
+// | 2    | 1    | new rotation state                            | uint8      |
+// | 2    | 1    | old rotation state                            | uint8      |
```

Comment:
How is it aligned at 0, 2 and 2? Please clarify the intent here. Is it 0, 1, 2? Or is it 0, 1, 1 and the states are only 4 bits? if so please use uint4 instead of uint8.